### DISCUSSION TITLE
Reminder

### DISCUSSION THREAD

**Discussion by frustak** [frustak/koal#18] [Ideas]
A place to add all things you want to do but have no particular goal or time in mind

**Comment by Glyphack** [comment]
I think this should still be a todo. Right now we have 3 different entities:
- Todo
- Goal
- DayFocus

But this one is a special kind of Todo. It does not have any particular goal.

We can add a new attribute to Todo object, `type`, and set it to a value for this particular todo.
I think the type `not_urget` might be a good name for them.

**Reply by Glyphack** [reply]
Server Change:

- Create Todo API will change to add the type.
- All methods returning Todo objects will change to return this field.
- This field is optional(It is going to be empty for the current todos)
